modules/python/init.zsh

```diff
@@ -34,6 +34,51 @@ if (( ! $+commands[python] && ! $+commands[pyenv] )); then
   return 1
 fi
 
+function _python-workon-cwd {
+    if [[ -z "$WORKON_CWD" ]]; then
```

Comment:
https://github.com/jayusa123/oh-my-zsh/commit/69116fa806f5a7405d33cc1f7c66d4c26ad4253b
> Avoid infinite `cd` loops under certain conditions.

Not sure if this is still relevant